packages/salesforcedx-vscode-core/test/vscode-integration/commands/auth/forceAuthLogout.test.ts

```diff
@@ -5,7 +5,7 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
```

Comment:
Note these tests need to be updated to account for the underlying core changes, but I just started by commenting out the Alias that has been removed and wanted to see how to the tests would fail.